gremlin/src/test/java/com/arcadedb/server/gremlin/AbstractGremlinServerIT.java

```diff
@@ -55,7 +55,7 @@ public void setTestConfiguration() {
           FileUtils.readStreamAsString(getClass().getClassLoader().getResourceAsStream("gremlin-server.groovy"), "utf8"));
 
       GlobalConfiguration.SERVER_PLUGINS.setValue("GremlinServer:com.arcadedb.server.gremlin.GremlinServerPlugin");
-
+      GlobalConfiguration.TYPE_DEFAULT_BUCKETS.setValue(8);
```

Comment:
Why do you need this?

Reply:
let me remove this and open another PR for IT tests